src/ol/renderer/webgl/PointsLayer.js

```diff
@@ -420,7 +419,7 @@ class WebGLPointsLayerRenderer extends WebGLLayerRenderer {
     const message = {
       type: WebGLWorkerMessageType.GENERATE_BUFFERS,
       renderInstructions: this.renderInstructions_.buffer,
-      useShortIndices: !includes(WEBGL_EXTENSIONS, 'OES_element_index_uint')
+      useShortIndices: !this.helper.hasOESElementIndexUint
```

Comment:
Good idea, although I'd look for an actual API on the helper side, could be something specific to this extension such as `this.helper.isElementIndexUintAllowed()` or something.

Reply:
And we should move the function out of the WebGLHelper class so it can be used without an instance

Reply:
Then if not on an instance the original implementation is fine IMO. The `WEBGL EXTENSIONS` array contains all available extensions, so:
- either we assume that *available extensions are always enabled on webgl contexts* in which case we only need to check against this array
- or we do not do this assumption and we check *on the helper instance* whether the extension was enabled

I went for the assumption initially, considering that there is no reason to not enable an extension in my knowledge. Anyway, keep in mind that there are dozens of extensions so writing specialized wrappers for each may become too verbose.